lib/unit_tests_utils/helper_utils.rb

```diff
@@ -0,0 +1,28 @@
+class UnitTestsUtils::HelperUtils
```

Comment:
Please pick a more appropriate name instead of a general one.
For example:
```ruby
class UnitTestsUtils::Manifest::Traverser
attr_reader :manifest
def initialize(manifest)
  @manifest = manifest
end

def find(path)
# your implementation...
end

end
```

Reply:
Also this way we might be able to add ```def valid_path?(path)``` more easily (even if it's private in the beginning) and are not overwhelmed with too many methods spamming he HelperUtils class.